doc/whats_new.rst

```diff
@@ -22,7 +22,7 @@ Current
 Changelog
 ~~~~~~~~~
 
-- no entries yet
+- :func:`read_raw_bids` to read electrodes.tsv only if all non-bad channels have coordinates; warns if not-bad ch and 'n/a' coordinate, by `Adam Li`_ (`#393 <https://github.com/mne-tools/mne-bids/pull/393>`_)
```

Comment:
don't use abbreviations here. This is user facing.

ch -> channels
not-bad -> good

etc.

also this should go into bugfix section